File: Selenium/Project/CRMActivity2.java
package examples;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedCondition;
import org.openqa.selenium.support.ui.WebDriverWait;

import java.time.Duration;

public class CRMActivity2 {

    public static void main (String[] args){

        //webdriver setup

        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm/");

        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(30));
        WebElement headImgUrl = driver.findElement(By.xpath("//img[@alt='SuiteCRM']"));
        System.out.println("header url is:" +headImgUrl.getAttribute("href"));

        WebElement headImgUrl1 = driver.findElement(By.xpath("//a[@title='SuiteCRM']"));
        System.out.println("header url is:" +headImgUrl1.getAttribute("href"));

        driver.close();





    }
}

File: Selenium/Project/CRM_Activity2.py
# Get the url of the header image
# Goal: Print the url of the header image to the console
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:

    driver.get("https://alchemy.hguy.co/crm/")

#  Get the url of the header image.
#  Print the url to the console.
    headerImage = driver.find_element(By.XPATH, "//div/img")
    actualUrl = headerImage.get_attribute("src")
    print("The url in Image is : " + actualUrl)
# Close the browser
    driver.quit()

